hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -126,9 +121,7 @@ public HoodieWriteClient(JavaSparkContext jsc, HoodieWriteConfig clientConfig,
   @VisibleForTesting
   HoodieWriteClient(JavaSparkContext jsc, HoodieWriteConfig clientConfig,
```

Comment:
Do all methods require a embedded server to be triggered ? For example, does bulkInsert need it ?